File: xijinhui_window/XiJinHuiWindow/Models/SetInvoice.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace XiJinHuiWindow.Models
{
    public class SetInvoice
    {
        [JsonPropertyName("id")]
        public int? Id { get; set; }

        [JsonPropertyName("invoice_date")]
        public string InvoiceDate { get; set; }

        [JsonPropertyName("invoice_start")]
        public int? InvoiceStart { get; set; }

        [JsonPropertyName("invoice_count")]
        public int? InvoiceCount { get; set; }

        [JsonPropertyName("invoice_end")]
        public int? InvoiceEnd { get; set; }

        [JsonPropertyName("created_at")]
        public string CreatedAt { get; set; }

        [JsonPropertyName("updated_at")]
        public string UpdatedAt { get; set; }

    }
}

File: xijinhui_laravel/app/Models/SetInvoice.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class SetInvoice extends Model
{
    use HasFactory;

    protected $fillable = [
        "invoice_date",
        "invoice_start",
        "invoice_count",
        "invoice_end",
    ];
}
